Tasks/NuGetInstaller/task.json

```diff
@@ -27,7 +27,15 @@
             "label": "Path to Solution",
```

Comment:
You'll need to bump the patch version in the file, build, and check-in the task.loc.json file also.